tests/evm/test_begin_tx.py

```diff
@@ -158,6 +158,9 @@ def test_begin_tx(tx: Transaction, callee: Account, result: bool):
                     0,
                 ),
                 (16, RW.Read, RWTableTag.CallContext, 1, CallContextFieldTag.IsStatic, 0, 0, 0),
+                (17, RW.Read, RWTableTag.CallContext, 1, CallContextFieldTag.LastCalleeId, 0, 0, 0),
+                (18, RW.Read, RWTableTag.CallContext, 1, CallContextFieldTag.LastCalleeReturnDataOffset, 0, 0, 0),
+                (19, RW.Read, RWTableTag.CallContext, 1, CallContextFieldTag.LastCalleeReturnDataLength, 0, 0, 0),
```

Comment:
The current conflict comes because of https://github.com/appliedzkp/zkevm-specs/pull/94
These tuples now have 10 elements, so they require 2 extra trailing zeroes.